### PR TITLE
Javascript2 week3

### PR DESCRIPTION
**Pull request by natata08**
This pull request includes:
- add inputs for 2 player names and a button to start a quiz;
- add two buttons [correct] and [wrong] with a certain behaviour
- add the number inputs for the player points;
- add playing a sound and ending the quiz game under certain conditions

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -302,3 +306,138 @@ function searchQuestions() {
...
+  input.addEventListener('input', checkInputs)
+);
+
+//start game
+let playersData = [];
+
+function startQuiz(event) {
+  event.preventDefault();
+
+  Array.from(playerNameInputs).forEach((input) => {
```
Comment (babak-f) [inline script.js]: As you will learn in JavaScript 3, this block can be simplified using the map function.

File: styles.css [hunk 2]
```diff
@@ -76,6 +77,18 @@ html {
   font-size: 62.5%; /* 10px */
 }
 
+@media (max-width: 1000px) {
+  html {
+    font-size: 60%;
```
Comment (babak-f) [inline styles.css:82]: Usually the best approach for setting font size is using em.

**Review by babak-f — APPROVED** [review]
Great job :)

File: script.js (hunk 1 above)
Comment (natata08) [reply script.js]: Good idea! I'll fix it.

**Comment by natata08** [comment]
@babak-f thank you for your review!

**Pull request merged**